monster-cards/src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>dojo2 Monster Cards</title>
+	</head>
+	<body>
+		<script src="./../../node_modules/dojo-loader/dist/umd/loader.min.js"></script>
+		<script src="./index.js"></script>
+			<header is="app-projector">
+				<div>
+					<dojo-widget data-options='{"id":"header-1"}'>
```

Comment:
Ids are mandatory? If so, how about a convention of [page]-[widget], e.g. home-header, shoppingCart-footer?